### PR TITLE
e2e: add more etcdctl tests

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/4259.

### CODE REVIEW

File: e2e/etcdctl_test.go [hunk 1]
```diff
@@ -60,13 +60,159 @@ func etcdctlSet(epc *etcdProcessCluster, key, value string, noSync bool) error {
...
+	if proxies := epc.proxies(); len(proxies) != 0 {
+		endpoint = proxies[0].cfg.acurl.String()
+	} else if backends := epc.backends(); len(backends) != 0 {
+		endpoint = backends[0].cfg.acurl.String()
+	}
+
+	args := []string{"../bin/etcdctl", "--endpoint", endpoint}
+	if noSync {
+		args = append(args, "--no-sync")
+	}
```
Comment (heyitsanthony) [inline e2e/etcdctl_test.go]: this is repeated for every etcdctl function; fold into `etcdctlPrefixArgs` like `cURLPrefixArgs`

**Comment by heyitsanthony** [comment]
overall looks OK

File: e2e/etcdctl_test.go (hunk 1 above)
Comment (gyuho) [inline e2e/etcdctl_test.go]: Ok I will simplify them. Thanks

**Comment by gyuho** [comment]
@heyitsanthony Just fixed. PTAL. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks!

**Pull request merged**